### PR TITLE
allow request location when needed

### PR DESCRIPTION
**Pull request by PeterPetrik**

### CODE REVIEW

File: app/qml/SettingsPanel.qml [hunk 1]
```diff
@@ -80,45 +81,27 @@ Page {
                 bold: true
             }
 
+            PanelItem {
+                height: settingsPanel.rowHeight
+                width: parent.width
+                color: InputStyle.clrPanelMain
+                text: qsTr("Enable Location")
```
Comment (wonder-sk) [inline app/qml/SettingsPanel.qml:88]: How about "Use GPS Location"

File: qgsquick/from_qgis/qgsquickpositionkit.h [hunk 2]
```diff
@@ -196,6 +204,12 @@ class QUICK_EXPORT QgsQuickPositionKit : public QObject
      */
     Q_INVOKABLE void useGpsLocation();
 
+    //! \copydoc QgsQuickPositionKit::enabled
+    bool isEnabled() const;
+
+    //! \copydoc QgsQuickPositionKit::enabled
+    void setIsEnabled( bool isEnabled );
```
Comment (wonder-sk) [inline qgsquick/from_qgis/qgsquickpositionkit.h:211]:
```suggestion
    void setEnabled( bool enabled );
```

**Review by wonder-sk — COMMENTED** [review]
Looks good.

What if Record button is pressed with location off? Previously it would center at GPS location. Does it do anything now? Or centers to invalid coordinates?

**Comment by PeterPetrik** [comment]
related to https://github.com/lutraconsulting/input/issues/953

**Review by wonder-sk — APPROVED** [review]

File: app/qml/SettingsPanel.qml [hunk 3]
```diff
@@ -80,45 +81,27 @@ Page {
                 bold: true
             }
 
+            PanelItem {
+                height: settingsPanel.rowHeight
+                width: parent.width
+                color: InputStyle.clrPanelMain
+                text: qsTr("Use GPS Location")
```
Comment (wonder-sk) [inline app/qml/SettingsPanel.qml:88]:
```suggestion
                text: qsTr("Use GPS location")
```
... for consistency with other settings

**Pull request merged**